app/src/main/java/com/mayokunadeniyi/instantweather/data/model/Sys.kt

```diff
@@ -0,0 +1,12 @@
+package com.mayokunadeniyi.instantweather.data.model
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * Created by haydar on 12/05/2020.
+ */
+@Parcelize
+data class Sys(
+    val country: String
+) : Parcelable
```

Comment:
Hi @HayDar-Android , I would love to suggest the use of better class names. We would want to keep the code self-explanatory well enough that just looking at the class name could give the reader if not all, but a little insight on what it is really for.